.gitpod/setup-ddev.sh

```diff
@@ -43,6 +43,4 @@ services:
 COMPOSEEND
 
 # Misc housekeeping before start
-ddev config global --router-bind-all-interfaces
-
-yes | ddev start
```

Comment:
I think you'll want to keep the `yes | ddev start` so it doesn't always say "You have a new ddev version, want to poweroff"